payment-handler/app-change-payment-method.js

```diff
@@ -0,0 +1,24 @@
+self.addEventListener('canmakepayment', event => {
+  event.respondWith(true);
+});
+
+self.addEventListener('paymentrequest', event => {
```

Comment:
Alternative, using async/await... might be little easier to read/debug if things go bad (hopefully didn't screw anything up refactoring it 🤞):
```JS
self.addEventListener("canmakepayment", event => {
  event.respondWith(true);
});

async function responder(event) {
  const methodName = event.methodData[0].supportedMethods;
  if (!event.changePaymentMethod) {
    return {
      methodName,
      changePaymentMethodReturned:
        "The changePaymentMethod() method is not implemented.",
    };
  }
  let changePaymentMethodReturned;
  try {
    const response = await event.changePaymentMethod(methodName, {
      country: "US",
    });
    changePaymentMethodReturned = response;
  } catch (err) {
    changePaymentMethodReturned = error.message;
  }
  return { methodName, changePaymentMethodReturned };
}

self.addEventListener("paymentrequest", event => {
  event.respondWith(responder(event));
});
```

Reply:
Thank you! Applied your suggestion, @marcoscaceres . Could you review the pull request again, please?